### ISSUE TITLE
Allow connection to additional WiFi network with internet access, while still providing hotspot

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-builder-rpi#34]
and pass through internet to the user's computer...?

I believe I've done this myself manually on a Pi, but it seems complex. I do believe that the Pi Zero W wireless card will allow this, however.

**Comment by dogi** [comment]
yes it works - we did pure wifi bridging in https://github.com/treehouses/cli
